test/extended/apibinding/apibinding.go

```diff
@@ -0,0 +1,44 @@
+package apibinding
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[sig-apibinding]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-apibinding")
+	)
```

Comment:
@Phaow can we change the name to kcp-sharedcomputewrkspace ?

Reply:
Hmm from my understanding, we'll also add new cases which use a new created compute resource test apibinding, so maybe the `kcp-sharedcomputeworkspace` not suitable. How about keep it fistly to see if Ke has any other better ideas as he is the feature owner of api.

Reply:
My understanding was this workspace is used for creating shared compute apibinding and schedule workloads on the same, I am okay to keep it this way but we do not have too many apibindings also right ? In our tests we only create one apibinding that is for shared compute.

Reply:
@Phaow i think i am okay with keeping it this way, thanks !!